lib/crypto/sr25519/sr25519.go

```diff
@@ -195,7 +212,7 @@ func (kp *Keypair) Private() crypto.PrivateKey {
 }
 
 // VrfSign creates a VRF output and proof from a message and private key
-func (kp *Keypair) VrfSign(t *merlin.Transcript) ([VRFOutputLength]byte, [VRFProofLength]byte, error) {
+func (kp *Keypair) VrfSign(t *merlin.Transcript) ([32]byte, [64]byte, error) {
```

Comment:
Why not use the constants here?

Maybe redefine them in a shared /lib/crypto/constants package?
Alternatively, please name the returns as

```suggestion
func (kp *Keypair) VrfSign(t *merlin.Transcript) (
	vfrOutput [32]byte, vfrProof [64]byte, err error) {
```

To serve as documentation, thanks!!

Reply:
why is this changed? please leave it the same as before

Reply:
I always forget that we can document the code like this, it really improves understanding, appreciate. 
I'm gonna rollback to the constants, looks more clear and they are used in another files, so a shared pkg seems useful

Reply:
rollback done in 7da55ab! @noot @qdm12